README.md

```diff
@@ -16,6 +17,15 @@ Pass a string (eg. a task or note content) to `MarkupParser#getMarkupEntries` an
 
 An optional list of flags can be included to limit the types of markup to be searched.
 
+To get emojis use `EmojiParser`:
+```
+Matcher matcher = EmojiParser.getEmojiPattern().matcher(string);
+while (matcher.find(start)) {
+    String emoji = EmojiParser.getEmoji(matcher.group(1));
+    ...
+}
+```
+
```

Comment:
As explained before, it would be great if `EmojiParser` was invisible to clients of this API to improve consistency and simplify the API. We wouldn't need this extra documentation in that case.